common/src/main/java/org/conscrypt/TrustManagerImpl.java

```diff
@@ -1011,15 +1011,15 @@ public boolean verify(String hostname, SSLSession session) {
         }
     }
 
-    private ConscryptHostnameVerifier getHttpsVerifier() {
+    private ConscryptHostnameVerifier getHttpsVerifier() { // doggo
         if (hostnameVerifier != null) {
             return hostnameVerifier;
         }
-        ConscryptHostnameVerifier defaultVerifier = getDefaultHostnameVerifier();
+        ConscryptHostnameVerifier defaultVerifier = getDefaultHostnameVerifier(); // this is still needed for current tests. Will I have to remove this and update teh tests to work with default or are teh predefined tests still valid?
         if (defaultVerifier != null) {
             return defaultVerifier;
         }
-        return GlobalHostnameVerifierAdapter.INSTANCE;
+        return Platform.getDefaultHostnameVerifier();
```

Comment:
Can `GlobalHostnameVerifierAdapter` be removed now?

Reply:
yeah definitely. I have to add a lot of tests still and some other changes to clean it up so that should all be updated by this week